Project.toml

```diff
@@ -1,6 +1,12 @@
 name = "ClimaComms"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-authors = ["CliMA Contributors <clima-software@caltech.edu>"]
+authors = [
+    "Kiran Pamnany <clima-software@caltech.edu>",
+    "Simon Byrne <simonbyrne@caltech.edu>",
+    "Charles Kawczynski <clima-software@caltech.edu>",
+    "Sriharsha Kandala <clima-software@caltech.edu>",
```

Comment:
@sriharshakandala what email would you like me to use?

Reply:
Sriharsha.kvs@gmail.com . Thanks.

Reply:
```suggestion
    "Sriharsha Kandala <Sriharsha.kvs@gmail.com>",
```